package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "alchemy-sdk",
-  "version": "2.0.4",
+  "version": "2.0.5-rc1",
```

Comment:
Can you undo this change?

Reply:
Hm I already published this to npm, should we keep it like this to reflect that?